source/guides/mattermost-ai-copilot-admin-guide.rst

```diff
@@ -0,0 +1,74 @@
+Admin Guide: Mattermost AI Copilot Plugin
```

Comment:
For consistency with the end user guide can we just call this "Admin Guide: Mattermost AI Copilot"

The guide mentions it's a plugin so I don't think it's necessary in the title

Reply:
Good call, missed this one first time around.